server/planning/feeding_services/event_http_service.py

```diff
@@ -87,7 +87,7 @@ def _update(self, provider, update):
         logger.info('Ingesting: %s', str(response.content))
 
         if isinstance(parser, NTBEventXMLFeedParser):
-            xml = ElementTree.parse(response.content)
+            xml = ET.fromstring(response.content)
```

Comment:
@thnkloud9 I assumed here that content is a text, instead of a file because I don't understand how this could be a file. And mainly because it's more convenient with the new test ^^